src/app/components/task/base/user-identity/user-identity-picker.component.ts

```diff
@@ -14,23 +14,23 @@ interface UserOption {
 
 const defaultUsers = [
     {
-        label: "Task user",
-        scope: AutoUserScope.task,
+        label: "Pool user",
```

Comment:
are you doing it this way to change the order?

Reply:
Yea figured default should be first? Plus then its alphabetical too which is always a plus :)

Reply:
Yeah make sense, so was the default already pool user? I don't see that changed.

Reply:
The default was 0 in this list, so just changing order and default.

Reply:
sounds good, just fix the test and all good